README.md

```diff
@@ -54,6 +54,14 @@ status are checked every 15 minutes, you can do it with this configuration:
 0/15 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bin/rails decidim_vocdoni:change_election_status > /dev/null
 ```
 
+When using dynamic census (when we the census can change during the election vote period), it is necessary to update it in to the Vocdoni blockchain. In order not to overflow the Vocdoni API by sending census updates one by one, this process is done periodically by sending all the users accumulated during that period. A common scenario for this is when you select an authorization method in order to be able to vote and people dynamically authorize themselves in order to vote. This introduces a waiting period that can be reduced to about a minute if these batch update are send every minute in a cron job.
+
+To configure Vocdoni to automatically send batch updates, add the following line to your crontab. This will allow updates to be sent every minute, reducing the waiting time for participants to a maximum of 70 seconds.
+
+```crontab
+* * * * * cd /home/user/decidim_application && RAILS_ENV=production bin/rails decidim_vocdoni:send_batch_updates > /dev/null
+```
+
```

Comment:
Lets use this
```suggestion
## Dynamic censuses

When using dynamic census (when we the census can change during the election vote period), it is necessary to update it in to the Vocdoni blockchain. 
A common scenario for this is when you want the internal Decidim userbase to be the census. Then,an authorization method is selected in order to be able to vote and, people dynamically authorize themselves in order to vote (because they might not have the proper authorization in the moment of voting).

This process can have a cost of a few tokens so it is not done automatically. The admin is responsible to update the census in the Vocdoni blockchain in the admin's monitoring page. The process of updating the census effectively removes the current census and creates a new one for the ongoing election, so it is advisable to do it with care and properly inform the voters at which time slots this will performed.

> In summary, the admin is responsible to update the census at all times, this is not an automatic process. Also, remember that this might have a monetary cost in terms of tokens.
```